rpcs3/Emu/System.cpp

```diff
@@ -1857,8 +2208,23 @@ void Emulator::GracefulShutdown(bool allow_autoexit, bool async_op)
 	}
 }
 
-void Emulator::Kill(bool allow_autoexit)
+extern bool try_lock_vdec_context_creation();
+
+void Emulator::Kill(bool allow_autoexit, bool savestate)
 {
+	if (!try_lock_vdec_context_creation())
+	{
+		sys_log.error("Failed to savestate: HLE VDEC (video decoder) context(s) exist."
```

Comment:
a savestate is not a verb

Reply:
This is an ever-evolving language, we can make it a verb.